spaceship/spec/connect_api/tunes/tunes_stubbing.rb

```diff
@@ -23,6 +23,43 @@ def stub_get_end_user_license_agreement
         stub_request(:get, "https://appstoreconnect.apple.com/iris/v1/apps/123/endUserLicenseAgreement").
           to_return(status: 200, body: read_fixture_file('end_user_license_agreement.json'), headers: { 'Content-Type' => 'application/json' })
       end
+
+      def stub_create_review_submission
+        stub_request(:post, "https://appstoreconnect.apple.com/iris/v1/reviewSubmissions").
+          to_return(status: 200, body: read_fixture_file('review_submission_created.json'), headers: { 'Content-Type' => 'application/json' })
+      end
+
+      def stub_cancel_review_submission
+        stub_request(:patch, "https://appstoreconnect.apple.com/iris/v1/reviewSubmissions/123456789").
+          with(body: { data: WebMock::API.hash_including({ attributes: { canceled: true } }) }).
+          to_return(status: 200, body: read_fixture_file('review_submission_cancelled.json'), headers: { 'Content-Type' => 'application/json' })
+      end
+
+      def stub_get_review_submission
+        stub_request(:get, "https://appstoreconnect.apple.com/iris/v1/reviewSubmissions/123456789").
+          to_return(status: 200, body: read_fixture_file('review_submission.json'), headers: { 'Content-Type' => 'application/json' })
+      end
+
+      def stub_get_review_submissions
+        stub_request(:get, "https://appstoreconnect.apple.com/iris/v1/apps/123456789-app/reviewSubmissions").
+          to_return(status: 200, body: read_fixture_file('review_submissions.json'), headers: { 'Content-Type' => 'application/json' })
+      end
+
+      def stub_submit_review_submission
+        stub_request(:patch, "https://appstoreconnect.apple.com/iris/v1/reviewSubmissions/123456789").
+          with(body: { data: WebMock::API.hash_including({ attributes: { submitted: true } }) }).
```

Comment:
If I write just plain `hash_including`, RubyMine tells me that there are two possible definitions: one coming from RSpec and the other from WebMock. It is probably some kind of mess regarding imports because in other places in the code, plain `hash_including` works.

I tried adding a `require` statement and directly referring to the RSpec version, but the tests kept on failing with `unrecognised method hash_including`. This instead works for some reason. 🤷 Again, just to know if someone asks.

Reply:
Ci sta, I also suspect something strange with the imports 🤔